token-metadata/program/tests/transfer.rs

```diff
@@ -637,6 +637,8 @@ mod auth_rules_transfer {
         .await
         .unwrap();
 
+        let original_token = nft.token.unwrap();
```

Comment:
Was this just a cleanup item or is there a technical reason it could not be unwrapped when the `TransferParams` was created?  I don't think its a problem, just curious if I'm not seeing if it is a fix for something or just making it neater.